cppcache/src/ThinClientRegion.cpp

```diff
@@ -2967,7 +2969,8 @@ void ThinClientRegion::executeFunction(
       } else if (err == GF_NOTCON) {
         attempt++;
         LOGDEBUG(
-            "ThinClientRegion::executeFunction with GF_NOTCON retry attempt = "
+            "ThinClientRegion::executeFunction with GF_NOTCON retry attempt "
```

Comment:
More odd formatting changes.

Reply:
Really wish I knew why this was happening.  It's as if `clang-format` v11 on Windows is not exactly the same as `clang-format` v11 on other platforms.  It's frustrating.

Reply:
That is even an odd one from clang-format on windows since it is splitting a sting 2 characters shorter. 🤷